### DISCUSSION TITLE
Option to add menu items that do nothing/have no commands?

### DISCUSSION THREAD

**Discussion by croak9854** [tomasklaen/uosc#585] [Q&A] [answered]
Is there an option to add menu items that do nothing but help me in remembering what certain keybinds do?
This is needed when using the script [input-event](https://github.com/natural-harmonia-gropius/input-event) and there are multiple _events_ set to the same key. I do not need those to work from the menu itself as it would be messy, but just to show in the menu. 
Example that would be okay:
```
# stuff  #! Speed > FF 3.5 & max   -   HOLD RIGHT
```
What should **stuff** be?

**Comment by tomasklaen** [comment]
We have `ignore` [hardcoded to be skipped](https://github.com/tomasklaen/uosc/blob/c8ad77a1a92d0667e1e66f11e84692cd03796ec8/scripts/uosc.lua#L241), and it makes sense to do that.

So maybe just add some `script-binding noop/noop` there, which won't do anything.

**Reply by croak9854** [reply]
Yes, this works. I have multiple keys to fast-forward at different speeds and wanted those to show in the menu. Thank you.

**Comment by hooke007** [comment, marked as answer]
command `print-text ""`

**Reply by croak9854** [reply]
This also works and seems simpler than the other answer. Thanks.
